app/javascript/helpers/PermissionChecker.js

```diff
@@ -0,0 +1,51 @@
+function hasManageUsers(user) {
```

Comment:
PermissionChecker will centralize the permissions checks and can be used throughout the  app to reduce boilerplate code (check the TODO comments added).